contracts/legacy/LegacyMakerManager.sol

```diff
@@ -554,11 +555,13 @@ contract LegacyMakerManager is Loan, BaseModule, RelayerModule, OnlyOwnerModule
      * @return The minimum amount of PETH to lock in the CDP
      */
     function minRequiredCollateral(bytes32 _cup, IMakerCdp _makerCdp) public returns (uint256 _minCollateral) {
-        _minCollateral = daiDebt(_cup, _makerCdp)    // DAI debt
-            .rmul(_makerCdp.vox().par())         // x ~1 USD/DAI
-            .rmul(_makerCdp.mat())               // x 1.5
-            .rmul(1010000000000000000000000000) // x (1+1%) cushion
-            .rdiv(_makerCdp.tag());              // ÷ ~170 USD/PETH
+        _minCollateral = rdiv(
+            rmul(
+                rmul(
+                    rmul(daiDebt(_cup, _makerCdp), _makerCdp.vox().par()), // DAI debt x ~1 USD/DAI
+                    _makerCdp.mat()),           // x 1.5
+                1010000000000000000000000000),  // x (1+1%) cushion
+            _makerCdp.tag());                  // ÷ ~170 USD/PETH
```

Comment:
What's the reason for avoiding the dot notation here and in `MakerV2Loan`? I find it less readable.

Reply:
Reason being is that while openzeppelin's `SafeMath` is a library, Maker's `DSMath` is a contract. Additionally the entire Maker solution uses this notation.